posts/RedHat-TelHai.md

```diff
@@ -0,0 +1,47 @@
+# רד האט מרחיבה את שיתוף הפעולה עם מכללת תל-חי
+
+<p dir="rtl">
+שי וטרסקו, Associate Manager, Software Engineering, Red Hat</p>
+
+
+<p dir="rtl">
```

Comment:
rather than wrapping in `<p dir="rtl">`, use frontmatter to set the lang and dir